### ISSUE TITLE
Enhancement: Draw lines based on Point A to Point B driving directions

### ISSUE THREAD

**Issue by nitrag** [mapbox/geojson.io#452]
It would be nice to have a feature where you can select two points on a map and then have the route (polyline) created automatically for you.

To Do:
- UI Icon to toggle mode
- UI Instructions
- click event grab Point A and Point B
- Google Maps API geojson decode
- Add polyline

I was thinking about planning a road trip where I create my own map with detours. For short trips I could create polylines manually but across a state....that would take hours and way to many clicks.

**Comment by tmcw** [comment]
Would love to, but there are no attribution/sharealike-free routers, so blocked by law.
